django_client_framework/api/base_model_api.py

```diff
@@ -111,8 +111,9 @@ def _order_queryset_by_param(self, queryset):
         Support generic filtering, eg: /products?_order_by=name
         """
         by = self.request.query_params.getlist("_order_by", ["pk"])
+        by_arr = by[0].split(",")
         try:
-            return queryset.order_by(*by)
+            return queryset.order_by(*by_arr)
         except Exception as execpt:
```

Comment:
For myself: changed due to design changes. The old API accepts `_order_by=pk&_order_by=anotherkey`. The new API accepts `_order_by=pk,anotherkey`.